tests/integration/standard/test_metadata.py

```diff
@@ -1357,20 +1357,31 @@ def make_function_kwargs(self, called_on_null=True):
                 'called_on_null_input': called_on_null}
 
     def test_functions_after_udt(self):
+        """
+        Test to to ensure udt's come after functions in in keyspace dump
+
+        test_functions_after_udt creates a basic function. Then queries that function and  make sure that in the results
+        that UDT's are listed before any corresponding functions, when we dump the keyspace
+        Ideally we would make a function that takes a udt type, but this presently fails because C* c059a56 requires
+        udt to be frozen to create, but does not store meta indicating frozen
+        SEE https://issues.apache.org/jira/browse/CASSANDRA-9186
+        Maybe update this after release
+        kwargs = self.make_function_kwargs()
+        kwargs['type_signature'][0] = "frozen<%s>" % udt_name
+        expected_meta = Function(**kwargs)
+        with self.VerifiedFunction(self, **kwargs):
+
+        @since 2.6.0
+        @jira_ticket PYTHON-211
+        @expected_result UDT's should come before any functions
+        @test_category function
+        """
+
         self.assertNotIn(self.function_name, self.keyspace_function_meta)
 
         udt_name = 'udtx'
         self.session.execute("CREATE TYPE %s (x int)" % udt_name)
 
-        # Ideally we would make a function that takes a udt type, but
-        # this presently fails because C* c059a56 requires udt to be frozen to create, but does not store meta indicating frozen
-        # https://issues.apache.org/jira/browse/CASSANDRA-9186
-        # Maybe update this after release
-        #kwargs = self.make_function_kwargs()
-        #kwargs['type_signature'][0] = "frozen<%s>" % udt_name
-
-        #expected_meta = Function(**kwargs)
-        #with self.VerifiedFunction(self, **kwargs):
```

Comment:
Moving this commented block into the header loses its original context in the code. Are we trying to have no inline comments?